Build Error due to mistake while merge on PR https://github.com/dsrvlabs/vatz/pull/514

### Checklist
- [x] Bug from Service (Vatz)
- [ ] Bug from Plugin (Vatz) 
- [ ] Malfunction on CI/CD 
- [ ] etc(e.g, Hardware version doesn't match, )

---

### **Describe the bug**
> A clear and concise description of what the bug is.


### **How to Reproduce**
> Steps to reproduce the behavior:

```
 make build   
go build -ldflags="-X 'github.com/dsrvlabs/vatz/utils.Version=main' -X 'github.com/dsrvlabs/vatz/utils.Commit=c2c6e6a'" -v
github.com/dsrvlabs/vatz/manager/healthcheck
github.com/dsrvlabs/vatz/manager/executor
# github.com/dsrvlabs/vatz/manager/healthcheck
manager/healthcheck/general_healthcheck.go:5:2: "errors" imported and not used
manager/healthcheck/general_healthcheck.go:81:20: undefined: fmt
make: *** [build] Error 1
 ✘ dongyookang DK 🍻   ~/ffplay/GolandProjects/xellos/vatz   main
 

```

### **Expected behavior**
> A clear and concise description of what you expected to happen.



### **Screenshots**
> If applicable, add screenshots to help explain your problem.

--- a/manager/healthcheck/general_healthcheck.go
+++ b/manager/healthcheck/general_healthcheck.go
@@ -2,18 +2,18 @@ package healthcheck
 
 import (
 	"context"
-	"errors"
-  "sync"
+	"fmt"
+	"sync"
 	"time"
-  
+
 	pb "github.com/dsrvlabs/vatz-proto/plugin/v1"
+	"github.com/dsrvlabs/vatz/manager/config"
 	dp "github.com/dsrvlabs/vatz/manager/dispatcher"
 	tp "github.com/dsrvlabs/vatz/manager/types"
-  "google.golang.org/protobuf/types/known/emptypb"
-  "github.com/dsrvlabs/vatz/manager/config"
 	"github.com/dsrvlabs/vatz/utils"
 	"github.com/robfig/cron/v3"
 	"github.com/rs/zerolog/log"
+	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 var (
@@ -70,9 +70,9 @@ func (h *healthChecker) PluginHealthCheck(ctx context.Context, gClient pb.Plugin
 		errorCount := 0
 		for _, dispatcher := range dispatchers {
 			sendNotificationError := dispatcher.SendNotification(deliverMSG)
-      if sendNotificationError != nil {
+			if sendNotificationError != nil {
 				log.Error().Str("module", "healthcheck").Msgf("failed to send notification: %v", err)
-        errorCount = errorCount + 1
+				errorCount = errorCount + 1
 			}
 		}
 

